patches/@react-native-community+netinfo+11.2.1+002+turbomodule.patch

```diff
@@ -2955,95 +2955,95 @@ index 878f7ba..0000000
 --- a/node_modules/@react-native-community/netinfo/windows/.npmignore
 +++ /dev/null
 @@ -1,92 +0,0 @@
--*AppPackages*
--*BundleArtifacts*
--
--#OS junk files
--[Tt]humbs.db
--*.DS_Store
--

```

Comment:
I can't figure out why this diff shows, I pasted the exact raw code from the prior commit and the code is identical...